### ISSUE TITLE
Some sentences don't seem to be getting indexed for searches

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#1865]
Some sentences don't seem to be getting indexed for searches.
Some sentences are missing from searches, but some sentences newer than the missing ones show up.

See the Wall.
https://tatoeba.org/eng/wall/show_message/31658#message_31658

In addition to the note on the Wall, I noticed this one, recently, too.

Search for Amastan's recently-contributed English sentences with "separatist."

https://tatoeba.org/eng/sentences/search?from=eng&to=ber&user=Amastan&sort=created&query=separatist

The top number is: 7836358
Kabyles don't need a fascist racist separatist group to defend them.
I'm sick of the separatist propaganda.
Ferhat Mehenni, the leader of a dangerous separatist group, has no right to speak in the name of all the Kabyles.
Kabylie, a Berber-speaking region in Algeria, doesn't need separatist terrorists.
The separatist flag is unwanted in Algerian marches.
etc.

You can see that there are actually the following more recent ones showing on this page
https://tatoeba.org/eng/Sentences/of_user/Amastan/eng

The bottom number is this: 7841405.
Those separatist shitheads are real sociopaths.
The separatists are sociopaths.
I have received new threatening messages from the separatists.
I have received new threats from the separatists.
I received threats from the separatists.
Ferhat Mehenni, an ex-singer, is now the "president" of a ridiculous but dangerous separatist movement.
Kabyles are traditionally a tight community and they don't like traitors. Therefore, the separatists have no future in our region.
We are not afraid of the separatists.
The separatists have developed an extremely violent political discourse lately.

**Comment by soliloquist-tatoeba** [comment]
I want to report a similar issue.

https://tatoeba.org/eng/sentences/show/2332801

There's something strange on this sentence's page. The names of the users on the comment and logs are not shown. I wonder how common this issue is and if it occurred after the code migration.

**Comment by ckjpn** [comment]
I think for an old sentences like this, it's not a current bug. In the past, I think when some members were banned from the site, and Sysko (maybe it was) deleted them, this happened.

**Comment by trang** [comment]
Not sure if related to the [crash that happened a few days ago](https://tatoeba.org/eng/wall/show_message/31668#message_31668).

Logs of that crash: https://gist.github.com/trang/c8fb097ecb905a247379b9967aece35f

**Comment by jiru** [comment]
I think it’s related.

I found out that the old sphinxsearch package left some files even after removal. There was a cron file /etc/cron.d/sphinxsearch that was starting a full index refresh as root everyday at midnight, effectively messing up with our own index refresh scripts. I had to use `apt-get remove --purge sphinxsearch` to get rid of these leftovers. I’m rebuilding the main indexes right now to add back sentences that havn’t been indexed. Hopefully this will prevent this problem from happening again.
